ext/opencensus_trace.c

```diff
@@ -250,7 +252,7 @@ static opencensus_trace_span_t *opencensus_trace_begin(zend_string *function_nam
         php_mt_srand(GENERATE_SEED());
     }
 #endif
-    span->span_id = (php_mt_rand() >> 1);
+    span->span_id = ((uint32_t) php_mt_rand()) >> 1;
```

Comment:
It should pass on 32-bit versions of PHP we now test against PHP 7.0 and 7.1 on a 32-bit debian build.
Also added comment about why this is in here.